pkg/env/costmodelenv.go

```diff
@@ -57,6 +57,7 @@ const (
 
 	UTCOffsetEnvVar = "UTC_OFFSET"
 
+	OutOfClusterEnabledEnvVar = "OUT_OF_CLUSTER_ENABLED"
```

Comment:
As in https://github.com/kubecost/cost-analyzer-helm-chart/pull/735 , we need to be a little more specific with naming to avoid confusion with our other ooc pipeline